### DISCUSSION TITLE
Support for Bun?

### DISCUSSION THREAD

**Discussion by onurhan1337** [unovue/shadcn-vue#790] [Q&A]
Firstly, thank you for your incredible work in building and maintaining this library for the Vue ecosystem.

I recently set up a Vue3 project using Bun.js, and I was excited to integrate shadcn-vue into the stack. Unfortunately, I noticed that the library does not currently support Bun.

I'm just wondering if there are any plans to introduce support for Bun soon. If so, could you let me know the progress and expected timeline for this feature? 

Additionally, if Bun support is not on the roadmap at the moment, could you suggest any potential workarounds or alternative libraries that would work well with a Bun + Vue3 setup?

Thank you for your time and support. Looking forward to your feedback!

**Comment by sadeghbarati** [comment]
https://github.com/unjs/nypm/releases/tag/v0.3.12

We using [`nypm`](https://github.com/unjs/nypm) for detecting packageManagers
In new release they added bun detection support we need to bump the version in shadcn-vue cli package
